lua/autorun/server/sv_property_restrictions.lua

```diff
@@ -0,0 +1,17 @@
+--/\-- Requiring / Including
+
+
+
+--/\-- Planning
+
+--/\-- Variables
+
+--/\-- Functions
+
+
+--/\-- Hooks
+hook.Remove("CanProperty", "persist_property")

```

Comment:
CFC has their own hook syntax, typically we do `CFC_<Project>_<Name>`

In this case, you might do:

```suggestion
hook.Remove( "CanProperty", "CFC_RestrictPersist" )
```

Side note, we always include spaces inside of our parenthesis. i.e.:
```lua
call( this, that )
-- vs
call(this, that)
```